src/main/java/org/prebid/server/bidder/adkernel/AdkernelBidder.java

```diff
@@ -79,9 +78,9 @@ private void processImp(Imp imp, Map<ExtImpAdkernel, List<Imp>> pubToImps) {
     }
 
     private static void validateImp(Imp imp) {
-        if (imp.getBanner() == null && imp.getVideo() == null) {
+        if (imp.getBanner() == null && imp.getVideo() == null && imp.getXNative() == null) {
             throw new PreBidException(String.format(
-                    "Invalid imp id=%s. Expected imp.banner or imp.video", imp.getId()));
+                    "Invalid imp id=%s. Expected imp.banner / imp.video / imp.native", imp.getId()));
```

Comment:
Can be simplified to `"Invalid imp id=" + imp.getId() + ". Expected imp.banner / imp.video / imp.native"`.